parse_Excel_loadTo_Sql.go

```diff
@@ -80,8 +56,8 @@ func main() {
 		log.Fatal(err)
 	}
 
-	fmt.Printf("Set to schema [%v]\n", config.GetSchema())
-	setSearchPath(db, config.GetSchema())
+	fmt.Printf("Set to schema [%v]\n", config.Schemago)
```

Comment:
`Schemago` looks wrong here. Surprised it even compiled for you

Reply:
Yep. Goo catch @marcesher.